src/table/table-role/grid-navigation.md

```diff
@@ -1,52 +1,46 @@
 # Table grid navigation
 
-Tables with interactive elements especially those featuring row selection or inline cell editing are assigned or can be assigned the role "grid" which implies extended keyboard navigation as per https://www.w3.org/WAI/ARIA/apg/patterns/grid.
+Tables with interactive elements require ARIA role "grid" and extended keyboard navigation as per https://www.w3.org/WAI/ARIA/apg/patterns/grid.
```

Comment:
nit: I know the doc is called "Table grid navigation" but it's probably good to clarify: "Tables that use grid navigation and have interactive elements [...]". (or something like that)